README.md

```diff
@@ -1,5 +1,26 @@
+# Windows Package Manager Repository
 
-# Contributing
+## Create a new private repository in Azure
+To simply the creation of a Windows Package Manager private repository, the `automation.ps1` [PowerShell script](Tools/Automation.ps1) is offered. This script will create new Elements required to host a Windows Package Manager private repository in Azure.
+
+The `automation.ps1` script has the following parameter inputs:
+| Required | Parameter          | Description                                                                                                                |
+|----------|--------------------|----------------------------------------------------------------------------------------------------------------------------|
+| Yes      | ResourcePrefix     | A string of letters which will be prefixed to your newly created Azure resources.                                          |
+| Yes      | Index              | A string of letters or numbers which will be sufixed to your newly created Azure resources.                                |
+| Yes      | AzResourceGroup    | The Resource Group that will be used to contain the Azure resources.                                                       |
+| No       | AzSubscriptionName | The name of the Azure Subscription that will be used to pay for the Azure resources.                                       |
```

Comment:
how is the subscription not required?

Reply:
The Subscription name is not required if the default (or only assigned) Subscription is the correct target Subscription. A Subscription is required to bill against, but does not need to be declared on script run.

Reply:
Can you add that to the doc? Because I was confused by the same thing. If you can add a link to how to set the default subscription, that would be great as well.

Reply:
Provided a similar comment to what I described above into the script identifying why this parameter is optional. Also included a link to setting Azure preferences